GainsTracker.UI/wwwroot/css/lightTheme.css

```diff
@@ -1,31 +1,32 @@
+    }
```

Comment:
We should one day probably find a better solution for these svg's, as this becomes quite a large file like this.  For now it'll be fine however.